src/knex-types/tbc_account_balance.ts

```diff
@@ -0,0 +1,6 @@
+export interface TBCAccountBalance {
```

Comment:
Thoughts on subfolders by functionality instead of having them all flat? Maybe too much unnecessary structure for now. e.g. 
`/knex-types/tbc/tbc_account.ts`
`/knex-types/tbc/tbc_account_balance.ts`
`/knex-types/token_account/token_account.ts`
`/knex-types/token_account/token_account_mint.ts` etc

Reply:
Yeah I had a similar thought. My gut said that subfolders wasn't necessary given the number of files we had right now, but I can definitely be convinced. I feel most strongly about having 1 file per DB table just because it makes the schema of the DB so much easier to grok quickly without needing to pull up `psql`